src/devices/tuya.ts

```diff
@@ -2282,7 +2282,8 @@ const definitions: Definition[] = [
             return {l1: 1, l2: 1};
         },
         whiteLabel: [
-            {vendor: 'Moes', model: 'MS-105B-M'},
+            tuya.whitelabel('Moes', 'MS-105B-M', '2 gang smart dimmer module', ['_TZE200_tsxpl0d0']),
```

Comment:
I expect that the `MS-105B-M` has a different manufname, so better to keep `{vendor: 'Moes', model: 'MS-105B-M'},` here

Reply:
Unfortunately, I did not find any indication of when to use [tuya.whitelabel](https://github.com/electronicm/zigbee-herdsman-converters/blob/919dbb15a3878309394d9c981c1a7fe5b0ef8856/src/lib/tuya.ts#L431-L437). So I took the line from `'KnockautX', 'FMD2C018'` and also rewrote `MS-105B-M` to standardize the code.
And now I have also seen my typo.

BTW: I have not found any indication that `MS-105B-M` is compatible with Zigbee. According to the manufacturer's website, datasheets, product pictures and manuals, it is a Wifi only device. Is it therefore not even out of place?

Reply:
I've removed it, thanks for checking.